eras/conway/impl/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Version history for `cardano-ledger-conway`
 
+## 1.7.1.0
```

Comment:
So, given that the actual code change is in master, does this mean that this changelog entry will  for 1.7.1.0 will never be past of master?

Reply:
Yes, whenever a release happens not from HEAD of master, there needs to be an adjustment to versions and changelog on master